.github/workflows/benchmark-command.yml

```diff
@@ -40,6 +44,7 @@ on:
           - 8h
           - 16h
           - 24h
+          - test101
```

Comment:
This might apply more to [`autogluon.bench`](https://github.com/autogluon/autogluon-bench/blob/master/sample_configs/amlb_configs/constraints.yaml), but should we make the constraint names more consistent with other AMLB constraints (e.g., `10m4c` for 10 minutes, 4 cores rather than `test101`)?

Reply:
Yes, sure I can make this change!